app/models/document.rb

```diff
@@ -23,7 +23,7 @@ class Document < ApplicationRecord
 
   has_and_belongs_to_many :localities
   belongs_to :document_category
-  belongs_to :building
+  belongs_to :building, optional: true
```

Comment:
try this, it keeps consistency with how buildings and people are related in the other models:
```
  has_and_belongs_to_many :buildings
  has_and_belongs_to_many :people
```